File: KPW_hs2020_Uebung4/Program.py
# -*- coding: utf-8 -*-
"""
Created on Tue Oct  6 12:18:14 2020

@author: urs
"""

factor = 10;


def max(a, b):
    """ 3 """
    if (a > b):
        return a
    else:
        return b


def compute(x):
    """ 2 """
    y = factor * max(x, 0);
    return y


def main():
    """ factor """
    n = 0
    n = int(input("Bitte Zahl eingeben"))
    print(compute(n))


main()

File: KPW_hs2020_Uebung4/Program.java
package ch.unibas.kpw;

public class Program {static long factor = 10;static long max(long a,
long b){/* 3 */if (a > b)return a;else return b;}static long compute(long x){/* 2 */long y=factor*max(x, 0);return y;}public static void main(String[] args){/* 1 */long n=0;java.util.Scanner sc=new java.util.Scanner(System.in);System.out.println("Bitte Zahl eingeben");n = sc.nextLong()
;System.out.println(compute(n));sc.close();}}